tensorflow/core/kernels/depthwise_conv_op.cc

```diff
@@ -296,8 +296,15 @@ class DepthwiseConv2dNativeOp : public BinaryOp<T> {
     // For in_depth == 1 and grouped convolutions.
     use_cudnn_ = CanUseCudnn() && std::is_same<Device, GPUDevice>::value;
     cudnn_use_autotune_ = CudnnUseAutotune();
-    use_cudnn_grouped_conv_ = false;
     dtype_ = DataTypeToEnum<T>::value;
+    // Use CuDNN grouped conv only when input/output is NCHW and float16(half).
+    // See cudnn release note 7.6.3. (https://docs.nvidia.com/deeplearning/sdk/c
+    // udnn-release-notes/rel_763.html#rel_763)
+#ifdef CUDNN_VERSION >= 7603
```

Comment:
I don't think this does the right thing: https://gcc.godbolt.org/z/zgwvF2

Reply:
You are right. Fixed.